samples/bmp280-sensor-azure-iot-hub/publish.bat

```diff
@@ -0,0 +1,4 @@
+dotnet publish -r linux-arm /p:ShowLinkerSizeComparison=true 
```

Comment:
I'd recommend to add this file somewhere in he readme instead

Reply:
Thanks for your comment @krwq, this .bat file is used by vscode tasks.json. What do you think, should I add this just in readme file?

Reply:
removed `publish.bat` and added into readme file as instruction.